specification/security/resource-manager/Microsoft.Security/stable/2018-06-01/security.json

```diff
@@ -0,0 +1,278 @@
+                }],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Pricing"
+                        }
+                      },                    
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "CloudError": {
+            "x-ms-external": true,
+            "type": "object",
+            "description": "Error response structure.",
+            "properties": {
+                "error": {
+                    "x-ms-client-flatten": true,
+                    "description": "Error data",
+                    "$ref": "#/definitions/CloudErrorBody"
+                }
+            }
+        },
+        "CloudErrorBody": {
+            "x-ms-external": true,
+            "type": "object",
+            "description": "Error details.",
+            "properties": {
+                "code": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+                },
+                "message": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "A message describing the error, intended to be suitable for display in a user interface."
+                }
+            }
+        },
+        "Resource": {
+            "type": "object",
+            "description": "Describes an Azure resource.",
+            "properties": {
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource Id"
+                },
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource name"
+                },
+                "type": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Resource type"
+                }
+            },
+            "x-ms-azure-resource": true
+        },
+        "PricingList": {
+            "description": "List of pricing configurations response",
+            "required": [
+                "value"
+            ],
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "description": "List of pricing configurations",
+                    "items": {
+                        "$ref": "#/definitions/Pricing"
+                    }

```

Comment:
do you support pagination?

Reply:
no, the number of returned items is limited.